httpcache.go

```diff
@@ -23,7 +25,9 @@ const (
 	fresh
 	transparent
 	// XFromCache is the header added to responses that are returned from the cache
-	XFromCache = "X-From-Cache"
+	XFromCache         = "X-From-Cache"
+	RANGESEPARATOR     = "-"
+	RANGETYPESEPARATOR = "="
```

Comment:
Use CamelCase for consistency with the stdlib (see http://www.reddit.com/r/golang/comments/2az1fz/convention_for_nonexported_constans/ for a discussion of this).